scipy/ndimage/interpolation.py

```diff
@@ -669,6 +657,30 @@ def rotate(input, angle, axes=(1, 0), reshape=True, output=None, order=3,
     rotate : ndarray
         The rotated input.
 
+    Examples
```

Comment:
I think the image size and the line breaks can be improved. Current rendering can be seen here https://12517-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.ndimage.rotate.html#scipy.ndimage.rotate

Reply:
OK, thank you for the review.